### DISCUSSION TITLE
Artisan script not included

### DISCUSSION THREAD

**Discussion by cretueusebiu** [orchestral/testbench#307] [Q&A]
Why is the [artisan](https://github.com/laravel/laravel/blob/8.x/artisan) script not included in the [Laravel app](https://github.com/orchestral/testbench-core/tree/master/laravel) ?

**Comment by crynobone** [comment]
Why not explain why you need `artisan` rather than asking why it isn't there.

**Reply by cretueusebiu** [reply]
I was writing a test for [spatie/laravel-schedule-monitor](https://github.com/spatie/laravel-schedule-monitor) where I needed to run an artisan command through the scheduler. 
The Laravel scheduler spawns a new process to run the commands, so it needs that `artisan` script in the base path of the application.

The test in question can be found [here](https://github.com/spatie/laravel-schedule-monitor/pull/43/files#diff-a3e4d61dd1b551a6040d0c2a5fcf6c485808b23a3bdd02976c36ef6150926236R170). We ended up including the artisan script in the package and copying to the testbech directory. 

Thanks 🙂
